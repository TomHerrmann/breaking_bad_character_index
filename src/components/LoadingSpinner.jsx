import React from 'react';

const LoadingSpinner = () => {
  return (
    // animated spinner source - https://loading.io/css/
    <div className="loading-spinner">
      <div></div>
      <div></div>
      <div></div>
      <div></div>
    </div>
  );
};

export default LoadingSpinner;
