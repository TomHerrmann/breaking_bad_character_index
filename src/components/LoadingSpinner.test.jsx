import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoadingSpinner from './LoadingSpinner';

describe('<LoadingSpinner />', () => {
  it('should render correctly', () => {
    const component = shallow(<LoadingSpinner />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
