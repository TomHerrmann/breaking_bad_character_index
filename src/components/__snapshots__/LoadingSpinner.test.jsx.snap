// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingSpinner /> should render correctly 1`] = `
<div
  className="loading-spinner"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
